src/main/java/org/broadinstitute/dsde/consent/ontology/datause/services/TextTranslationServiceImpl.java

```diff
@@ -20,22 +22,21 @@
 public class TextTranslationServiceImpl implements TextTranslationService {
 
     private OntologyTermSearchAPI api;
+    private static final Logger logger = LoggerFactory.getLogger(TextTranslationServiceImpl.class);
 
     // This is a cache, used to associate a type (element of
     // the set { "disease", "organization", "commercial-status" }) to each named class.
     // It's an expensive operation (we're going to have to use a reasoner!) so we want
     // to cache it here in this class and not use it again if we don't need to.
     private Map<String, String> namedClassTypes;
-    private OntModel model;
-
+    private Collection<URL> urls = new ArrayList<>();
+    private OntModel model = null;
```

Comment:
Nit: pretty sure this assignment is a no-op in Java because object variables are automatically initialized to `null`